src/utilities/corners.ts

```diff
@@ -12,64 +12,80 @@ export interface CornerData {
   cy: number;
 }
 
+// Expects angle input ranging from -PI to +PI
+function getCursorIndex(angle: number, idx: number) {
+  return (Math.floor(((17 * Math.PI) / 8 + angle) / (Math.PI / 4)) + idx) % 8;
```

Comment:
Do you have some notes about this formula? Can you put them in a comment?

Reply:
Added an explanation. SGTY?

Reply:
https://github.com/zakodium-oss/react-roi/pull/122/commits/04c23bcfb03c5c61b0f66faab76a412ac8f98acd

Reply:
SGTM